src/test/starknet.t.sol

```diff
@@ -124,6 +127,9 @@ contract StarknetTests is GoerliDssSpellTestBase, ConfigStarknet {
 
         assertEq(daiBridge.wards(addr.addr("MCD_PAUSE_PROXY")), 1, "StarknetTest/pause-proxy-not-ward-on-dai-bridge");
         assertEq(daiBridge.wards(addr.addr("MCD_ESM")),         1, "StarknetTest/esm-not-ward-on-dai-bridge");
+
+        uint256 l2DaiBridge = daiBridge.l2DaiBridge();
+        assertEq(daiBridge.l2DaiBridge(), starknetValues.l2_dai_bridge, "StarknetTest/wrong-l2-dai-bridge-on-dai-bridge");
```

Comment:
we should also test the dai approval here (unless I missed it elsewhere)

Reply:
Do you mean this: https://github.com/makerdao/spells-goerli/blob/cd1621bc49499e7982fc4bf8b011aa6438d064cc/src/test/starknet.t.sol#L114?

Reply:
Assuming that this test passes, this should be sufficient. Not a big fan of using `uint256(-1)` to specify a value when we have used `type(uint).max` elsewhere, but the initial concerns have been addressed so marking this as resolved.

Reply:
Yes, thank you @maciejka I missed that